app.js

```diff
@@ -0,0 +1,47 @@
+const express = require('express');
+var flash = require('connect-flash');
+var session = require('express-session');
+var passport = require('passport');
+var flash = require('connect-flash');
```

Comment:
duplicate import (flash imported twice), also if you use const, please use const for all imports and potentially variables, unless you need to re-assign them.